src/wasm/wasm-s-parser.cpp

```diff
@@ -219,6 +219,10 @@ void SExpressionParser::parseDebugLocation() {
   while (debugLocEnd[0] && debugLocEnd[0] != '\n') {
     debugLocEnd++;
   }
+  if (debugLocEnd == debugLoc) {
+    loc = nullptr;
+    return;
+  }
```

Comment:
How does this relate to the PR - is it necessary because of something, or necessary for a new test, or something else?

Reply:
This is just parsing `;;@` with nothing after.